src/components/dialog/dialog.component.js

```diff
@@ -67,7 +74,7 @@ class Dialog extends Modal {
   }
 
   centerDialog = (animating) => {
-    const height = this._dialog.offsetHeight / 2,
+    const height = this._dialog.scrollHeight / 2,
       width = this._dialog.offsetWidth / 2,
```

Comment:
Did you checked to see if offsetWidth was causing a similar issue in the horizonal axis?

Reply:
I didn't, I have to check it

Reply:
I have checked and it work and also after double check I revert offsetHeight because we don't need to change that to make it work